scripts/prebuild.ts

```diff
@@ -0,0 +1,63 @@
+import * as fs from 'fs';
+import * as path from 'path';
+import axios from 'axios';
+
+console.warn('Prebuilding...');
+
+const rootDir = path.dirname(__dirname);
+
+function idPoorPrebuild(country: string): void {
+  const dataDir = path.join(rootDir, `public/data/${country}`);
+  const file = path.join(dataDir, 'idpoor.json');
+
+  switch (country) {
+    case 'cambodia':
```

Comment:
There's a type in the frontend (`type Country`) which would be nice to use here. I have no clue if its possible however since it's in a completely different folder. All good with keeping this as a string if you don't see a way in using that.